builder/builder-next/worker/worker.go

```diff
@@ -50,7 +50,7 @@ import (
 )
 
 func init() {
-	version.Version = "v0.11.6"
+	version.Version = "v0.11.7-0.20230519102302-348e79dfed17"
```

Comment:
Is there any specific reason to backport a non-tagged commit?
Can we just release v0.11.7?

Reply:
I opened this one because there was a possible regression in Docker 24.0; I wasn't 100% sure if that only needed changes on the client-side (buildx), or also needed fixes on the daemon side.

We were preparing the 24.0.1 release, and there was no ETA yet for v0.11.7, so I prepared the vending updates. However, discussing with Jed and Tonis, it turned out that we didn't need changes on the daemon side, so I moved this one back to draft (and can probably close it until there's a v0.11.7 release); https://github.com/moby/moby/pull/45582#issuecomment-1554440486

Reply:
Updated this PR, because we found another issue that did result in a panic, https://github.com/docker/buildx/issues/1836